src/auto-advance.js

```diff
@@ -67,12 +67,27 @@ const setup = (player, delay) => {
     // we need to cancel the auto-advance. This could mean the user seeked
     // back into the content or restarted the content. This is reproducible
     // with an auto-advance > 0.
-    player.one('play', cancelOnPlay);
+    // player.one('play', cancelOnPlay);
```

Comment:
can someone help me understand the errors? I'm assuming it's because I commented this out